packages/shorebird_cli/lib/src/commands/patch/patch_ios_command.dart

```diff
@@ -307,12 +307,19 @@ Current Flutter Revision: $currentFlutterRevision
 
         final useLinker = AotTools.usesLinker(release.flutterRevision);
         if (useLinker) {
-          final exitCode = await _runLinker(
+          final (:exitCode, :linkPercentage) = await _runLinker(
             releaseArtifact: releaseArtifactFile,
           );
 
-          if (exitCode != ExitCode.success.code) {
-            return exitCode;
+          if (exitCode != ExitCode.success.code) return exitCode;
+
+          if (linkPercentage != null && linkPercentage < 50) {
```

Comment:
Pulling 50 out into a constant with docs (explaining things like "why 50", etc.) would probably be good